src/Appwrite/Utopia/Response/Model/Provider.php

```diff
@@ -15,36 +15,36 @@ class Provider extends Model
     public function __construct()
     {
         $this
-            ->addRule('key', [
+            ->addRule('$id', [
                 'type' => self::TYPE_STRING,
-                'description' => 'Provider.',
+                'description' => 'Provider ID.',
                 'default' => '',
-                'example' => 'github',
+                'example' => '5e5ea5c16897e',
             ])
             ->addRule('name', [
                 'type' => self::TYPE_STRING,
-                'description' => 'Provider name.',
+                'description' => 'The name for the provider instance.',
                 'default' => '',
-                'example' => 'GitHub',
+                'example' => 'Mailgun',
             ])
-            ->addRule('appId', [
+            ->addRule('provider', [
                 'type' => self::TYPE_STRING,
-                'description' => 'OAuth 2.0 application ID.',
+                'description' => 'The name of the provider service.',
                 'default' => '',
-                'example' => '259125845563242502',
+                'example' => 'mailgun',
             ])
-            ->addRule('secret', [
-                'type' => self::TYPE_STRING,
-                'description' => 'OAuth 2.0 application secret. Might be JSON string if provider requires extra configuration.',
-                'default' => '',
-                'example' => 'Bpw_g9c2TGXxfgLshDbSaL8tsCcqgczQ',
-            ])
-            ->addRule('enabled', [
```

Comment:
Thinking about it now, we might an `enabled` attribute on `provider` 🤔 we should be able to disable a providers use without entirely deleting it, same as we allow for auth providers.

Reply:
added enabled attribute in providers collection and added in model